tests/pauli/test_pauli_utils.py

```diff
@@ -418,38 +437,21 @@ def test_qwc_complement_adj_matrix_exception(self):
         with pytest.raises(ValueError, match="Expected a binary array, instead got"):
             qwc_complement_adj_matrix(not_binary_observables)
 
-    @pytest.mark.parametrize(
-        "pauli_word,wire_map,expected_string",
-        [
-            (PauliX(0), {0: 0}, "X"),
-            (Identity(0), {0: 0}, "I"),
-            (PauliZ(0) @ PauliY(1), {0: 0, 1: 1}, "ZY"),
-            (PauliX(1), {0: 0, 1: 1}, "IX"),
-            (PauliX(1), None, "X"),
-            (PauliX(1), {1: 0, 0: 1}, "XI"),
-            (PauliZ("a") @ PauliY("b") @ PauliZ("d"), {"a": 0, "b": 1, "c": 2, "d": 3}, "ZYIZ"),
-            (PauliZ("a") @ PauliY("b") @ PauliZ("d"), None, "ZYZ"),
-            (PauliX("a") @ PauliY("b") @ PauliZ("d"), {"d": 0, "c": 1, "b": 2, "a": 3}, "ZIYX"),
-            (4.5 * PauliX(0), {0: 0}, "X"),
-            (qml.prod(PauliX(0), PauliY(1)), {0: 0, 1: 1}, "XY"),
-            (PauliX(0) @ PauliZ(0), {0: 0}, "Y"),
-            (3 * PauliZ(0) @ PauliY(3), {0: 0, 3: 1}, "ZY"),
-            (qml.s_prod(8, qml.PauliX(0) @ qml.PauliZ(1)), {0: 0, 1: 1}, "XZ"),
-        ],
-    )
+    PAULI_WORD_STRINGS = _make_pauli_word_strings()
+
+    @pytest.mark.parametrize("pauli_word,wire_map,expected_string", PAULI_WORD_STRINGS)
     def test_pauli_word_to_string(self, pauli_word, wire_map, expected_string):
         """Test that Pauli words are correctly converted into strings."""
         obtained_string = pauli_word_to_string(pauli_word, wire_map)
         assert obtained_string == expected_string
 
+    with qml.operation.disable_new_opmath_cm():
+        PAULI_WORD_STRINGS_LEGACY = _make_pauli_word_strings()
```

Comment:
Doing this instead of using `convert_to_legacy_H` because new op math and legacy op math might have different ordering of pauli strings.

Reply:
Curious 🤔